its/src/test/java/com/sonar/it/csharp/AutoGeneratedTest.java

```diff
@@ -33,61 +33,62 @@
 
 public class AutoGeneratedTest {
 
-    @ClassRule
-    public static TemporaryFolder temp = TestUtils.createTempFolder();
+  @ClassRule
```

Comment:
sorry for the whitespace change noise, you can enable "hide whitespace changes" in GH for easy review